10.detect-separation/scripts/nbconverted/0.download-validation-data.py

```diff
@@ -0,0 +1,202 @@
+
+# coding: utf-8
+
+# # Download and Process Neuroblastoma RNAseq Data
+# 
+# **Gregory Way 2019**
+# 
+# We are downloading the dataset associated with [Harenza et al. 2017](https://doi.org/10.1038/sdata.2017.33). The data profiles RNAseq data from 39 commonly used neuroblastoma (NBL) cell lines.
+# 
+# We are interested in the MYCN amplification status of these cell lines. We will test if the MYCN amplification score learned through the BioBombe signature approach applied to TARGET data generalizes to this cell line dataset.
+
+# In[1]:
+
+
+import os
+import requests
+import pandas as pd
+from urllib.request import urlretrieve
+
+from sklearn import preprocessing
+
+
+# In[2]:
+
+
+url = "https://ndownloader.figshare.com/files/14138792"
+name = "2019-01-22-CellLineSTAR-fpkm-2pass_matrix.txt"
+path = os.path.join("download", name)
+
+
+# In[3]:
+
+
+os.makedirs("download", exist_ok=True)
+
+
+# In[4]:
+
+
+urlretrieve(url, path)
+
+
+# In[5]:
+
+
+get_ipython().system(' md5sum "download/2019-01-22-CellLineSTAR-fpkm-2pass_matrix.txt"')
+
+
+# ## Download Phenotype Data
```

Comment:
Maybe this is not necessary, I would just clarify that by phenotype you mean "MYCN status".  Not sure if you want to have the specific criteria that is used to describe what is meant by "amplified"